app/modules/utils.py

```diff
@@ -30,6 +30,30 @@ def fail_on_missing_static_folder(
         )
 
 
+def is_valid_sex(value):
+    return value is None or value in {'male', 'female', 'unknown'}
+
+
+def is_valid_latitude(value):
+    if not isinstance(value, float):
+        return False
+    max_latitude = 90.0
+    min_latitude = -90.0
+
+    # Validate range
+    return min_latitude <= value <= max_latitude
```

Comment:
Note that boolean expression chaining is a feature of Python 3: https://docs.python.org/2/reference/expressions.html#comparisons

Reply:
I'm presuming you are not expecting any change from that comment as we are not going to be using python 2 at any point.